.github/workflows/language-reference.yaml

```diff
@@ -0,0 +1,61 @@
+name: language-reference-documentation
+
+on:
+  push:
+    branches:
+      - 'language-reference-stable'
+  pull_request:
+    branches:
+      - 'language-reference-stable'
+
+jobs:
+  build-and-push:
+    if: github.event_name == 'push'
+    runs-on: ubuntu-latest
+    steps:
+      - name: Get current date
+        id: date
+        run: echo "::set-output name=date::$(date +'%Y-%m-%d')"
+
+      - name: Git Checkout
+        uses: actions/checkout@v3
+
+      - name: Cache Coursier
+        uses: actions/cache@v1
+        with:
+          path: ~/.cache/coursier
+          key: sbt-coursier-cache
+
+      - name: Cache SBT
+        uses: actions/cache@v1
+        with:
+          path: ~/.sbt
+          key: sbt-${{ hashFiles('**/build.sbt') }}
+
+      - name: Set up JDK 11
+        uses: actions/setup-java@v1
+        with:
+          java-version: 11
+
+      - name: Generate reference documentation and test links
+        run: |
+          ./project/scripts/sbt scaladoc/generateReferenceDocumentation
+          ./project/scripts/docsLinksStability ./scaladoc/output/reference ./project/scripts/expected-links/reference-expected-links.txt
+
+      - name: Push changes to scala3-reference-docs
+        if: github.event_name == 'push'
+        uses: actions/checkout@v3
+        with:
+          repository: lampepfl/scala3-reference-docs
+          fetch-depth: 0
+          submodules: true
+          ssh-key: ${{ secrets.DOCS_DEPLOY_KEY }}
+          path: 'scala3-reference-docs'
+        run: |
+          \cp -a scaladoc/output/reference/. scala3-reference-docs/
```

Comment:
Is the leading `\` a typo?

Reply:
No, actually it prevents shell from using an alias: cp="cp -i" which is present in modern Linux distributions. It makes the cp command to not overwrite existing files by default and ask user in case of conflict. We want to always overwrite.